samples/highcharts/series-heatmap/inactive-except/demo.js

```diff
@@ -0,0 +1,102 @@
+Highcharts.chart('container', {
+
+    chart: {
+        type: 'heatmap',
+        animation: false,
+        events: {
+            load: function () {
+                var chart = this;
+                function change() {
+                    chart.series[0].update({
+                        states: {
+                            inactive: {
+                                except: this.value
+                            }
+                        }
+                    });
+                }
+
+                document
+                    .querySelectorAll('input[name="inactive-except"]')
+                    .forEach(function (radio) {
```

Comment:
IE doesn't support `forEach` on a `NodeList` element ([source](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach)). Previously it was resolved like in this code: https://github.com/highcharts/highcharts/blob/974b67d60923373f6b5d4abb565e62dc811e1111/ts/modules/oldie.src.ts#L1948